posthog.go

```diff
@@ -730,6 +730,15 @@ func (c *client) getFeatureFlagFromDecide(key string, distinctId string, groups
 		return nil, err
 	}
 
+	// Check if feature_flags is quota limited
+	if decideResponse.QuotaLimited != nil {
+		for _, limitedFeature := range *decideResponse.QuotaLimited {
+			if limitedFeature == "feature_flags" {
+				return false, nil
+			}
+		}
+	}
+
```

Comment:
Looks like this code is duplicated in a couple places. Perhaps a helper method `isQuotaLimited(decideResponse)` (or something like that) would make sense here?